fili-core/src/main/java/com/yahoo/bard/webservice/web/handlers/PartialDataRequestHandler.java

```diff
@@ -35,23 +30,19 @@
     public static final String PARTIAL_DATA_HEADER = "partialData";
 
     protected final @NotNull DataRequestHandler next;
-    protected final @NotNull PhysicalTableDictionary physicalTableDictionary;
     protected final @NotNull PartialDataHandler partialDataHandler;
 
     /**
      * Wrap the response processor in a partial data check.
      *
      * @param next The next request handler to invoke
-     * @param physicalTableDictionary  the repository of slice data
-     * @param partialDataHandler the PartialDataHandler to use
+     * @param partialDataHandler the service to calculate partial data from table availabilities
      */
     public PartialDataRequestHandler(
             DataRequestHandler next,
-            PhysicalTableDictionary physicalTableDictionary,
```

Comment:
Need to deprecate constructor, not just change it. This will definitely break people w/o letting them migrate smoothly